Week3/homework/project/tipCalculator.js

```diff
@@ -0,0 +1,26 @@
+const btn = document.getElementById('btn');
+const billAmount = document.getElementById('billAmount');
+const service = document.getElementById('service');
+const peopleSharing = document.getElementById('peopleSharing');
+const tipAmount = document.getElementById('tipAmount');
+const each = document.getElementById('each');
```

Comment:
It's a good practice to use the variable names in order to distinguish the variables between them.
For example I try to name html elements using the El suffix. eg billEl or tipEl
This will help me later to use tip in order to name the actual tip variable and not `tip_amount`. And also I would know that tipEl refers to the html element and can be used only to get or set things to html